SWLOR.Game.Server/Service/Stat.cs

```diff
@@ -718,6 +748,34 @@ public static void ClearNPCDefense()
         /// <returns>A modified defense value.</returns>
         private static int CalculateEffectDefense(uint creature, int defense)
         {
+
+
+            { 
+                var willBonus = GetAbilityModifier(AbilityType.Willpower, creature); 
+                            
+             if (StatusEffect.HasStatusEffect(creature, StatusEffectType.Tutaminis1))
+             defense += 10 + (willBonus * 1);
```

Comment:
Lines immediately after if statements like this should be tabbed over to signify they're underneath the if statement.